src/backend/kvm/vm/mod.rs

```diff
@@ -24,22 +22,31 @@ use kvm_ioctls::{Kvm, VmFd};
 use lset::Span;
 use mmarinus::{perms, Kind, Map};
 use primordial::Page;
-use x86_64::VirtAddr;
 
+#[cfg(target_arch = "x86_64")]
+use x86_64::{PhysAddr, VirtAddr};
+
+use std::collections::VecDeque;
 use std::marker::PhantomData;
 use std::num::NonZeroUsize;
 use std::sync::{Arc, RwLock};
 
-pub struct Vm<A: Arch, P: Personality> {
+pub struct Vm<P: Personality> {
     kvm: Kvm,
     fd: VmFd,
     regions: Vec<Region>,
     syscall_blocks: Span<VirtAddr, NonZeroUsize>,
-    arch: A,
     _personality: PhantomData<P>,
+    cpus: VecDeque<u64>,
+
+    #[cfg(target_arch = "x86_64")]
+    rip: PhysAddr,
```

Comment:
This is actually a guest `VirtAddr`.
```suggestion
    rip: VirtAddr,
```

Reply:
ok, I don't care anymore, because it's going to be ripped out soon :)